http/1.7/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -699,6 +700,227 @@ In the *Configuration XML* editor, the `tls:context`, `tls:trust-store` and `tls
     </tls:context>
 ----
 
+[[oauth2-grant-authentication-connected-apps]]
+== Configure OAuth2 Authorization Code Grant Type Authentication Using Connected Apps
+:page-aliases: connectors::http/http-configure-authorization-code-for-requester.adoc
+
+You can also configure the OAuth2 authorization code grant type authentication using the xref:access-management::connected-apps-developers.adoc[Connected Apps] feature.  
+
+The following example shows how to configure the OAuth2 authorization code grant type authentication for the HTTP *Request* operation by creating a Mule app to access a protected resource and the resource in the resource server. See xref:mule-runtime::authorization-grant-types.adoc[OAuth Authorization Grant Types]. The example covers how to:
+
+* Create the Mule app
```

Comment:
Should these be numbered? (are they sequential?)

Reply:
Is this correct - a protected resource and the resource in the resource server?

Reply:
They are sequential. I'll change it to enumeration. :)